src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -407,6 +408,28 @@ private void createMesosTask(Offer offer, Request request) {
     LOGGER.info("Launching task " + taskId.getValue() + " with URI " +
                 joinPaths(jenkinsMaster, SLAVE_JAR_URI_SUFFIX));
 
+    for (final Computer computer : Jenkins.getInstance().getComputers()) {
```

Comment:
Sorry for not being clear. Yes, the id of the mesos task is the same as the name of the Jenkins slave and this was done to make it easy to figure out which mesos task to kill when the jenkins slave goes idle.

What I am confused about is that, this code suggests we are creating duplicate task ids which is unexpected. Can you elaborate what do you mean by your second sentence?

Thanks.